src/frontends/onnx/frontend/src/op/col2im.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/op/col2im.hpp"
+
+#include "core/operator_set.hpp"
+using namespace ov::op;
+
+namespace ov {
+namespace frontend {
+namespace onnx {
+namespace ai_onnx {
+namespace opset_1 {
+ov::OutputVector col2im(const ov::frontend::onnx::Node& node) {
+    const auto data = node.get_ov_inputs().at(0);
+    const auto output_shape = node.get_ov_inputs().at(1);
+    
```

Comment:
Please assign the node.get_ov_inputs() to a variable istead calling it several times.
Also, looks like the third input (at 2) "block_shape" is not used 
https://onnx.ai/onnx/operators/onnx__Col2Im.html#col2im-18